server/src/main/java/io/crate/planner/optimizer/iterative/GroupReference.java

```diff
@@ -52,12 +52,14 @@ public class GroupReference implements LogicalPlan {
 
     private final int groupId;
     private final List<Symbol> outputs;
+    private final Set<RelationName> relationNames;
     private static final String ERROR_MESSAGE =
         "Operation is not supported in GroupReference, it needs to be resolved to it's referenced LogicalPlan";
 
-    public GroupReference(int groupId, List<Symbol> outputs) {
+    public GroupReference(int groupId, List<Symbol> outputs, Set<RelationName> relationNames) {
```

Comment:
Could change this to `Collection<RelationName>` if we're using `Set.copyOf` anyways

Reply:
👍